src/components/elementRenderer/elementRenderer.tsx

```diff
@@ -11,13 +11,31 @@ interface ElementRendererProps {
 const ElementRenderer = (props: ElementRendererProps) => {
   const MaybeElement = props.supportedElements[props.message.format]
 
+  function extractFormat(updateFormat: string) {
+    const lettersInUpdate = 'update'.length
+    const format = updateFormat
+      .substring(0, updateFormat.length - lettersInUpdate)
+      .trim()
+
+    return format
+  }
+
+  const updateMessages = props.message.threadMessages?.filter(
+    (threadMessage) =>
+      threadMessage.sender === props.message.sender &&
+      threadMessage.format.includes('Update') &&
```

Comment:
do we want to check for `update` in the beginning or end? maybe pattern matching and capturing would be more efficient here

Reply:
Beginning probably makes most sense